progress_test.go

```diff
@@ -43,7 +43,7 @@ func TestProgress_FromScratch(t *testing.T) {
 func TestProgress_ParallelComplete(t *testing.T) {
 	tmp := t.TempDir()
 	name := filepath.Join(tmp, "chunk.zip")
-	p, err := newProgress(name, "https://test.etc/chunk.zip", 5, 2048, false)
+	p, err := newProgress(name, tmp, "https://test.etc/chunk.zip", 5, 2048, false)
```

Comment:
`os.TempDir()` here and bellow.

Reply:
What's the advantage of using `t.TempDir()` instead of `os.TempDir()`? I thought that `t.TempDir()` would be nicer because it removes the directory after the test (if I remember it correctly)

Reply:
My bad. I overlooked that assignment. It is all good.